view/src/cg/group4/view/screen/RewardScreen.java

```diff
@@ -103,6 +103,7 @@ protected void fillDisplay(final Collection collection) {
         DecimalFormat format = new DecimalFormat("0.##");
         java.util.List<Collectible> list = collection.sort(new RarityComparator());
         cContentTable.clear();
+        System.out.println(list);
```

Comment:
Is the Syso here needed? Maybe try to either use log function or just remove it.